web/src/js/pages/multiplayer.js

```diff
@@ -11,15 +11,6 @@ let currentGameId = null;
 
 var currentEvents = [];
 
-function escapeHtml(unsafe) {
```

Comment:
Duplicate to existing function https://github.com/osuAkatsuki/hanayo/blob/html-escape-usernames/web/src/js/akatsuki_src.js#L832